src/app/components/BuildPreviewBanner/index.tsx

```diff
@@ -22,5 +27,19 @@ export const BuildPreviewBanner: FC = () => {
     return null
   }
 
-  return <StyledAlert severity="warning">{t('buildPreview')}</StyledAlert>
+  return (
+    <StyledAlert severity="warning">
+      {t('buildPreview')}
+      <br />
+      {t('buildPreviewFeedback')}
+      <Link
+        component={RouterLink}
+        to="https://forms.gle/456sjk9VXhhKkVz19"
```

Comment:
externalLinks

Reply:
I kept old message for build-preview deploys
> Please note this is an experimental build of Oasis Explorer and that data that is shown might be incorrect.

and added new message for build-internal deploys
> Please note this is an internal launch meant to gather your feedback. Please share your feedback with screenshots through this from: https://forms.gle/456sjk9VXhhKkVz19